Client/Frontend/Contextual Hint/ContextualHintViewModel.swift

```diff
@@ -16,6 +16,10 @@ enum ContextualHintViewType: String {
     case jumpBackIn = "JumpBackIn"
     case inactiveTabs = "InactiveTabs"
     case toolbarLocation = "ToolbarLocation"
+
+    var hintExtraForToolbarLocation: String {
+        BrowserViewController.foregroundBVC().isBottomSearchBar ? "ToolbarLocationBottom" : "ToolbarLocationBottom"
```

Comment:
uhhhhhh, they're both bottom? :P

```suggestion
        BrowserViewController.foregroundBVC().isBottomSearchBar ? "ToolbarLocationBottom" : "ToolbarLocationTop"
```